src/widgets/search-box/search-box.js

```diff
@@ -168,6 +185,10 @@ function searchBox({
         return actualEvent => actualEvent.keyCode === expectedKeyCode && fnToExecute(actualEvent);
       }
 
+      function getQueryAndCall(fnToExecute) {
```

Comment:
fnToExecute could be reduced to `f` or `fn`. Also those functional helpers :100: could be moved somewhere else.